doc/user/content/ingest-data/mysql/azure-db.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Ingest data from Azure DB for MySQL"
+description: "How to stream data from Azure DB for MySQL to Materialize"
+menu:
+  main:
+    parent: "mysql"
+    name: "Azure DB"
+    indentifier: "mysql-azure-db"
+---
+
+{{< private-preview />}}
+
+This page shows you how to stream data from [Azure DB for MySQL](https://azure.microsoft.com/en-us/products/MySQL)
+to Materialize using the [MySQL source](/sql/create-source/mysql/).
+
+## Before you begin
+
+{{% mysql-direct/before-you-begin %}}
+
+## Step 1. Enable GTID-based binlog replication
+
+{{< note >}}
+GTID-based replication is supported for Azure DB for MySQL [flexible server](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/overview-single).
+It is **not supported** for single server databases.
+{{</ note >}}
+
+Before creating a source in Materialize, you **must** configure Azure DB for
+MySQL for GTID-based binlog replication. This requires enabling binlog replication and
+the following additional configuration changes:
+
+Configuration parameter          | Value  | Details
+---------------------------------|--------| -------------------------------
+`log_bin`                        | `ON`   |
+`binlog_format`                  | `ROW`  | This configuration is [deprecated as of MySQL 8.0.34](https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_format). Newer versions of MySQL default to row-based logging.
+`binlog_row_image`               | `FULL` |
+`gtid_mode`                      | `ON`   |
+`enforce_gtid_consistency`       | `ON`   |
+`replica_preserve_commit_order`  | `ON`   | Only required when connecting Materialize to a read-replica for replication, rather than the primary server.
+
```

Comment:
Unlike the previous table, this one does not contain entry
```
`binlog retention hours`         | 168    |
```

Is this one not necessary or not that relevant here?
If it is, should we move this configuration to a separate page to have a single definition of it (assuming the values are the same for all usages)?